#message for triggering object detection of a specific object
#contains object type to trigger corresponding node and region of interest to constrain search

#object type, reference to the object in the database
string obj_type 

#image region of interest used to constrain object detection
pr2_pick_perception/ROI2d region2d 
